label_studio/projects/models.py

```diff
@@ -1018,6 +1018,21 @@ def get_all_export_storage_objects(self):
 
         return storage_objects
 
+    @cached_property
+    def multipage_labeling_values(self):
```

Comment:
speaking of locality, can this be defined in LSE in the indicator where it's actually used? afaict it only depends on the label config, it should be a pure use of the SDK LabelConfig object, and the reason to put it here is just for cached_property? Does this actually cause a significant performance problem?

Reply:
1. It requires access to `parsed_label_config` column of the Project model, which is LSO part
2. SDK Label Config is not ready to handle this as it requires recomputations for every project instance, which is inefficient
3. No significant bottlenecks, but caching is only to ensure the property is computed once (project update() would retrieve and build a new instance)